### PR TITLE
remove iptables cleanup from local-up-cluster.sh

### PR DESCRIPTION
**Pull request by atiratree**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Docker iptables usage was removed in https://github.com/kubernetes/kubernetes/pull/106909 so there is no need for iptables cleanup. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -139,7 +139,7 @@ if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
 fi
 
 if [ "$(id -u)" != "0" ]; then
-    echo "WARNING : This script MAY be run as root for docker socket / iptables functionality; if failures occur, retry as root." 2>&1
+    echo "WARNING : This script MAY be run as root for docker socket functionality; if failures occur, retry as root." 2>&1
 fi
```
Comment (adisky) [inline hack/local-up-cluster.sh:143]: We can remove this entire line, from what i have tested, the script is working without root

Comment (atiratree) [reply hack/local-up-cluster.sh]: +1, I have tested this as well and we can do without the warning

Comment (dims) [reply hack/local-up-cluster.sh]: sounds good @adisky

**Review by adisky — COMMENTED** [review]
/skip
/lgtm

**Comment by adisky** [comment]
/sig node
(don't know which sig this script comes under, added node as changes are dockershim removal followup)

**Pull request merged**